### DISCUSSION TITLE
Keypoint-Detection for LabGym?

### DISCUSSION THREAD

**Discussion by yasminetani** [umyelab/LabGym#242] [Q&A]
Good afternoon. Does LabGym only support instance segmentation or does it also allow keypoint detection. I know that detectron2 does recognize keypoint detection JSON annotation files, but I am having some difficulty with labgym recognizing it. I am guessing it may be strictly instance segmentation - is that the case?

**Comment by yujiahu415** [comment]
> Good afternoon. Does LabGym only support instance segmentation or does it also allow keypoint detection. I know that detectron2 does recognize keypoint detection JSON annotation files, but I am having some difficulty with labgym recognizing it. I am guessing it may be strictly instance segmentation - is that the case?

Hi,
Unfortunately LabGym currently only supports instance segmentation not keypoint detection, because of the design of Categorizers.
